pkg/llm-d-inference-sim/simulator_test.go

```diff
@@ -132,17 +139,15 @@ var _ = Describe("Simulator", func() {
 			}
 
 			Expect(numberOfChunksWithUsage).To(Equal(1))
-			Expect(chunk.Usage.PromptTokens).To(Equal(int64(4)))
+			Expect(chunk.Usage.PromptTokens).To(Equal(userMsgTokens))
 			Expect(chunk.Usage.CompletionTokens).To(BeNumerically(">", 0))
 			Expect(chunk.Usage.TotalTokens).To(Equal(chunk.Usage.PromptTokens + chunk.Usage.CompletionTokens))
 
 			msg := strings.Join(tokens, "")
-			expectedMsg := userMessage
-			if mode == modeRandom {
-				expectedMsg = getFullTextFromPartialString(msg)
```

Comment:
Why were the comparisons of the randomly generated text removed? The comparison must now be done "sentence by sentence" in the generated code.

This comment  applies to many places in the code where getFullTextFromPartialString used to be invoked and was deleted.